spec/factories/auth.rb

```diff
@@ -29,4 +29,37 @@
 
     initialize_with { [uai, type, category, function, uaj, tna_sym, tty_code, tna_code].join("$") }
   end
+
+  # rubocop:disable Layout/LineLength
+
+  factory :freduresdel, class: "String" do
+    transient do
+      uai { "0380083J" }
+      tty_code { "LP" }
+      resp_attribute { "FrEduRneResp" }
+      applicationname { "aplypro" }
+    end
+
+    initialize_with do
+      "aplypro|/redirectionhub/redirect.jsp?applicationname=#{applicationname}|14/11/2023|31/12/9999|agerard1|#{resp_attribute}=#{uai}$UAJ$PU$N$T3$#{tty_code}$320|sconet-frontal1||"
```

Comment:
Quand j'essaye, le linter me fait :
```
FactoryBot/FactoryAssociationWithStrategy: Use an implicit, explicit or inline definition instead of hard coding a strategy for setting association within factory.
```